.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   test:
     strategy:
       matrix:
-        node: [ '0.8', '0.10', '0.12', '4', '6', '8', '10', '12', latest ]
+        node: [ '0.8', '0.10', '0.12', '4', '6', '8', '10', '12', '14', latest ]
```

Comment:
@kzc this addresses https://github.com/mishoo/UglifyJS/pull/4658#issuecomment-788075261